lib/r10k/action/runner.rb

```diff
@@ -68,14 +68,23 @@ def setup_settings
       end
 
       def setup_authorization
-        begin
-          license = R10K::Util::License.load
+        if token = @settings.dig(:forge, :authorization_token)
+          if forge_url = @settings.dig(:forge, :baseurl)
+            logger.debug "Using token '#{token}' to connect to '#{forge_url}'"
```

Comment:
Do we maybe not want to log the token? Seems like a potential security concern.

Reply:
yeah, right, left this in there by accident